examples/RunTemporaryCommand/runTempCommand.go

```diff
@@ -0,0 +1,291 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"regexp"
+	"time"
+
+	"github.com/TheJumpCloud/jcapi"
+)
+
+const (
+	COMMAND_NAME_RANDOM_PART int = 16
+
+	MAX_COMMAND_LEN int = 16384 // Maximum size of a command to send to JumpCloud
+
+	RESULT_POLL_TIME     int = 5  // Check every RESULT_POLL_TIME seconds for command results
+	RESULT_MAX_POLL_TIME int = 70 // Stop checking for results after RESULT_MAX_POLL_TIME_SECONDS (must be a minimum of 60 seconds)
+
+	URL_BASE string = "https://console.jumpcloud.com/api"
+)
+
+func makeImmediateCommand(name, command, commandType, shell, user string) (cmd jcapi.JCCommand) {
+	cmd = jcapi.JCCommand{
+		Name:        name,
+		Command:     command,
+		CommandType: commandType,
+		User:        user,
+		LaunchType:  "manual",
+		Schedule:    "immediate",
+		Timeout:     "0", // No timeout
+		ListensTo:   "",
+		Trigger:     "",
+		Sudo:        false,
+		Shell:       shell,
+		Skip:        0,
+		Limit:       10,
+	}
+
+	return
+}
```

Comment:
I think this would be a good example of a place where if one was okay with different method signatures, you could omit the named return parameter, and instead just `return jcapi.JCCommand{...}`.